autoload/rainbow_main.vim

```diff
@@ -83,7 +83,7 @@ fun rainbow_main#gen_config(ft)
 		return 0
 	else
 		let conf = extend(extend({'syn_name_prefix': substitute(a:ft, '\v\A+(\a)', '\u\1', 'g').'Rainbow'}, dft_conf), af_conf)
-		let conf.cycle = has('gui_running')? s:lcm(len(conf.guifgs), len(conf.guis)) : s:lcm(len(conf.ctermfgs), len(conf.cterm))
+		let conf.cycle = has('gui_running')? s:lcm(len(conf.guifgs), len(conf.guis)) : s:lcm(len(conf.ctermfgs), len(conf.cterms))
```

Comment:
Tested it locally. I can confirm, that it is broken before, and this fixes the issue.

Reply:
thanks